frontend/src/pages/projectPage/ProjectList.tsx

```diff
@@ -82,29 +85,41 @@ function ProjectList() {
       } catch (e) {
         console.error(e);
       }
+      try {
+        const uids = team.map((user) => user.uid);
+        const data = {
+          uids: uids,
+        };
+        const req = await api();
+        const res = await req.post(`project/${projectId}/invite-users`, data);
```

Comment:
The function is called create project, but then you are inviting users? I don't really get it...